### ISSUE TITLE
Opus timecode only changed every 8th packet

### ISSUE THREAD

**Issue by sneaker2** [mbunkus/mkvtoolnix#2084]
Hi,

when I mux Opus (Ogg) using mkvmerge (15.0.0)  I get the following timecodes:
```
# timecode format v2
0
0
0
0
0
0
0
0
159.98976
159.98976
159.98976
159.98976
159.98976
159.98976
159.98976
159.98976
320.000352
320.000352
320.000352
320.000352
320.000352
320.000352
320.000352
320.000352
479.990112
479.990112
479.990112
479.990112
479.990112
479.990112
479.990112
479.990112
...```

Seems to be put into the usual SimpleBlocks grouping 8 packets but without a default duration. Samples uploaded to ftp.

P.S.: IIRC Opus can be variable frame-length so we need to be careful about simply setting a default duration

**Comment by mbunkus** [comment]
This is simply how it works with `BlockGroup` elements. It's perfectly fine.

**Comment by sneaker2** [comment]
The specs are kinda confusing me. They say:

> There can be many Blocks in a BlockGroup provided they all have the same timecode. It is used with different parts of a frame with different priorities.

Since all the Opus frames should have different timecodes I didn't think this would apply.

For the record: I actually stumbled upon this when playing in VLC. Wheen you seek the playback starts, then there's a short drop-out and after that playback continues normally. When I added the appropriate default-duration this doesn't happen. ffmpeg muxes Opus with only one frame per SimpleBlock.
